static/src/javascripts/projects/commercial/modules/third-party-tags/audience-science-pql.js

```diff
@@ -28,36 +28,52 @@ define([
     }
 
     function onLoad() {
-        var asiPlacements = window.asiPlacements;
-        var segments = storage.local.get(storageKey) || {};
-        // override the global value with our previously stored one
-        window.asiPlacements = segments[section];
-        segments[section] = asiPlacements;
-        storage.local.set(storageKey, segments);
+        isLoaded = true;
+        mediator.emit('commercial:audience-science:loaded');
```

Comment:
I think i've missed something.  I'm not sure how the code guarantees that `pql` is loaded, and targeting has been set, before the dfp requests are sent out?